colossalai/inference/quant/smoothquant/models/base_model.py

```diff
@@ -22,53 +26,10 @@
 from ....tensor_parallel.kvcache_manager import MemoryManager
```

Comment:
Though not modifications made in this PR, maybe it's better to import from colossalai.inferece.XXX

Reply:
Updated.